cdhweb/people/templates/people/snippets/profile_card.html

```diff
@@ -12,28 +12,33 @@
         <div class="content">
             <p class="name">{{ profile.title }}</p> {# profile page title = person's name #}
             <div class="title">
-            {% with position=profile.user.positions.first %}
-                {% if show_cdh_position and position %} {# show position within the cdh #}
-                    <p>{% if not position.is_current %}{{ position.years }}{% if show_grant and profile.user.latest_grant %}{% else %}<br/>{% endif %}{% endif %}
-                    {# when showing both position and grant, if person has both omit line breaks #}
-                    {{ profile.user.positions.first.title|default:'' }}</p>
-                {% endif %}
-            {% endwith %}
             {# NOTE: right now, if someone has both grant and cdh staff role, both will show #}
             {% if show_grant %} {# display grant role #}
                 {% with grant=profile.user.latest_grant %}
                     {% if grant %}
                     {# NOTE: for unknown reasons, on qa these are not returned as date objects; handle either date or YYYY-MM-DD string#}
```

Comment:
just noticed this `NOTE`, kind of bizarre. smells like a mysql timezone thing maybe?

Reply:
@thatbudakguy ooh, maybe? that would be nice if we could resolve it. Although I feel like if it were the timezone issue it would be impacting other things too.